src/pages/cart/cart.component.ts

```diff
@@ -1,14 +1,26 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { TitleComponent } from "../../components/title/title.component";
 
+import { Product } from 'src/interfaces/productInterface';
+import { ShoppingCartService } from 'src/services/shopping-cart.service';
+import { ProductCardComponent } from "../../components/product-card/product-card.component";
+
 @Component({
     selector: 'nbl-cart',
     standalone: true,
     templateUrl: './cart.component.html',
     styleUrls: ['./cart.component.css'],
-    imports: [CommonModule, TitleComponent]
+    imports: [CommonModule, TitleComponent, ProductCardComponent]
 })
-export class CartComponent {
+export class CartComponent implements OnInit{
+    cartProducts : Product[] = [];
```

Comment:
Same idea. This will always be in the context of the CartComponent. 
```suggestion
    products: Product[] = [];
```